### PR TITLE
docs: script to add css mod variables to the corresponding markdown files

### PR DESCRIPTION
**Pull request by venkateshjo**
## How and where has this been tested?
 - **How this was tested:** 
 - **Browser(s) and OS(s) this was tested with:**

### CODE REVIEW

**Review by pfulton — APPROVED** [review]

File: scripts/mod-extractor.js [hunk 1]
```diff
@@ -26,42 +37,34 @@ function writeToFile(file) {
...
-                    writeToFile(readFile);
-                } 
-            });
-        });
+        }
     }
 }
 
-searchFilesForString(rootDir, regex, fileExtension);
+searchFilesForString(rootDir, pattern, fileExtension);
```
Comment (Rajdeepc) [inline scripts/mod-extractor.js:69]: How will you execute this script during the build process?

**Review by Rajdeepc — CHANGES_REQUESTED** [review]

**Comment by Rajdeepc** [comment]
@venkateshjo Can you also please make sure the description is well articulated, Jira tickets are mentioned, Images (if any) and also how to test this locally all commands should be noted in the description of the ticket. It would be easier for @pfulton  and @castastrophe to test this locally.
Thanks!!!

File: scripts/mod-extractor.js (hunk 1 above)
Comment (venkateshjo) [reply scripts/mod-extractor.js:69]: Currently, this script is not part of the build process. User can trigger the build with the following command
yarn docs:mod

Comment (Rajdeepc) [reply scripts/mod-extractor.js:69]: Okay Nice!!! But this also needs to go into the build process eventually. Also please think on that part on how you can include this inside the component-builder build process.

**Comment by venkateshjo** [comment]
Sure. 
JIRA: https://jira.corp.adobe.com/browse/CSS-306

The script can be triggered with the following command
"yarn docs:mod"

This will generate the corresponding markdown files if the script detects --mod variables in the .css files.

Screenshot 2023-02-21 at 8 54 36 PM

File: scripts/mod-extractor.js (hunk 1 above)
Comment (castastrophe) [reply scripts/mod-extractor.js:69]: @Rajdeepc True! Patrick & I had talked about making this manual for now until we can merge in your updates that way there is less likely to be a conflict.

**Review by castastrophe — APPROVED** [review]

**Pull request merged**